frontend/src/components/ExperimentCollection/ExperimentCollectionDashboard/ExperimentCollectionDashboard.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { API_ROOT } from "../../../config";
+import ExperimentCollection from "@/interfaces/ExperimentCollection.interface";
+
+
+interface ExperimentCollectionDashboardProps {
+    experimentCollection: ExperimentCollection;
+}
+
+export const ExperimentCollectionDashboard: React.FC<ExperimentCollectionDashboardProps> = ({ experimentCollection }) => {
+
+    const dashboard = experimentCollection?.dashboard || [];
+
+    // TODO: get next experiment and about link from experimentCollection
+    const nextExperiment = experimentCollection.next_experiment; // TODO: get next_experiment from experimentCollection
+    const aboutContent = experimentCollection.about_content;
+
+    return (
+        <>
+            <div className="hero">
+                <div className="intro">
+                    <p>{experimentCollection?.description}</p>
+                    <nav className="actions">
```

Comment:
Where has the redirect gone?

Reply:
It's in `ExperimentCollection.jsx` (this page's parent)